lib/queries/generate-referral-credits.js

```diff
@@ -2,8 +2,7 @@
 
 const async = require('async');
 const constants = require('../constants');
-
-const REFERRAL_CREDIT_TYPE = constants.CREDIT_TYPE.AUTO;
+const REFERRAL_CREDIT_TYPE = constants.CREDIT_TYPES.AUTO;
```

Comment:
This should work...when @dylanlott and I tested this route it generated the credits correctly.